components/usage/pkg/stripe/stripe_test.go

```diff
@@ -89,3 +91,15 @@ func TestCustomerQueriesForTeamIds_MultipleQueries(t *testing.T) {
 		})
 	}
 }
+
+func TestStartOfNextMonth(t *testing.T) {
+	now := time.Now()
+	currentYear, currentMonth, _ := now.Date()
+	firstOfMonth := time.Date(currentYear, currentMonth, 1, 0, 0, 0, 0, time.UTC)
+	nextMonth := firstOfMonth.AddDate(0, 1, 0).Month()
+
+	expectedStartOfNextMonth := time.Time(time.Date(currentYear, nextMonth, 1, 0, 0, 0, 0, time.UTC))
+	actualStartOfNextMonth := getStartOfNextMonth(now)
+
+	require.Equal(t, actualStartOfNextMonth, expectedStartOfNextMonth)
```

Comment:
This is mostly recreating the function you've implemented. A slightly better approach is to fix the date you're starting with, and write out what you expect.

```suggestion
	ts := time.Date(2022, 10, 10, 10, 10, 10, 0, time.UTC)
	
	require.Equal(t, time.Date(2022, 11, 1, 0, 0, 0, time.UTC), getStartOfNextMonth(ts))
```

Also, the order to `require.Equal` is `expected` first, `actual` second ;)